docs/annotate_edges.md

```diff
@@ -0,0 +1,34 @@
+# annotate (edges)
+
+_subclass of [annotate](./annotate.md)_
+
+This operation adds attributes to knowledge graph edges. Implementations of this operation are _unique_; their behavior is not fully-specified by the operation parameters.
+
+### examples
+
+- [input](../examples/annotation/messages/01_knowledge.json), [output](../examples/annotation/messages/04_annotated_edges.json)
+
+### input requirements
+
+None
+
+### output guarantees
+
+None
+
+### allowed changes
+
+- add attributes to kedges
+
+### parameters
+
+```yaml
+properties:
+  attributes:
```

Comment:
Do we wish to collect anywhere all the possible attributes that can be added? I imagine that in a future UI, it would be helpful to have a drop-down list that looks like:
1. Add all possible attributes
2. Add pmids 
3. Add clinical trial phase
4. Etc.

This will need discussion

Reply:
Yeah that would probably be really helpful. Maybe add an enum section in this property?

Reply:
@finnagin Let's discuss during todays meeting